firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -21,7 +21,7 @@
 // Components:
```

Comment:
We decided to remove these style header comments in today’s meeting, please update.